sql/core/src/test/scala/org/apache/spark/sql/execution/SparkPlanTest.scala

```diff
@@ -98,7 +100,7 @@ class SparkPlanTest extends SparkFunSuite {
       planFunction: Seq[SparkPlan] => SparkPlan,
       expectedAnswer: Seq[Row],
       sortAnswers: Boolean = true): Unit = {
-    SparkPlanTest.checkAnswer(input, planFunction, expectedAnswer, sortAnswers) match {
+    SparkPlanTest.checkAnswer(input, planFunction, expectedAnswer, sortAnswers, sqlContext) match {
```

Comment:
If `sqlContext` is already a member of SparkPlanTest, is it possible that we don't need to pass it to `checkAnswer`?